auth-web/package-lock.json

```diff
@@ -47,7 +47,7 @@
         "@types/vuelidate": "^0.7.13",
         "@typescript-eslint/eslint-plugin": "^5.23.0",
         "@typescript-eslint/parser": "^5.23.0",
-        "@volar-plugins/vetur": "*",
+        "@volar-plugins/vetur": "latest",
```

Comment:
If you rebase then you should not see this change.

Also, why the jump from 1.1.28 to 1.1.30?

Reply:
I changed 1.1.30 to 1.1.29. This "latest" shows up when I do "npm i".

Reply:
OK. Looks like that package was changed in package.json but not package lock (until now).